frontend/packages/console-app/src/components/cloud-shell/CloudShellTab.tsx

```diff
@@ -9,8 +10,9 @@ import './CloudShellTab.scss';
 const CloudShellTab: React.FC = () => {
   const { t } = useTranslation();
   const devWorkspaceFlag = useFlag(FLAG_DEVWORKSPACE);
+  const mchFLag = useFlag(MCH_AVAILABILITY_FLAG);
 
-  if (devWorkspaceFlag === false) return <Redirect to="/" />;
+  if (devWorkspaceFlag === false || mchFLag === true) return <Redirect to="/" />;
```

Comment:
mch flag could take some time to be available but we shouldn't be showing the console while console in the meantime. We need to wait for the flag to be ready.

Reply:
noted